riscv/src/runtime.rs

```diff
@@ -376,7 +376,7 @@ impl Runtime {
     }
     pub fn with_arith(mut self) -> Self {
         self.add_submachine(
-            "std::machines::arith::Arith",
+            "std::machines::arith48::Arith48",
```

Comment:
Should all `Arith` machines be in the same `arith` module?

Reply:
Yea i was thinking about putting multiple machines in the same module, but i think readability gets worse. Fine with either, with a slight preference to 1 machine per file